pkg/controller/certificaterequests/venafi/venafi_test.go

```diff
@@ -525,27 +536,59 @@ func TestSign(t *testing.T) {
 								Type:               cmapi.CertificateRequestConditionReady,
 								Status:             cmmeta.ConditionFalse,
 								Reason:             cmapi.CertificateRequestReasonFailed,
-								Message:            "Failed to obtain venafi certificate: this is an error",
+								Message:            "Failed to request venafi certificate: this is an error",
 								LastTransitionTime: &metaFixedClockStart,
 							}),
 							gen.SetCertificateRequestFailureTime(metaFixedClockStart),
 						),
 					)),
 				},
 			},
-			fakeSecretLister: failGetSecretLister,
-			fakeClient:       clientReturnsGenericError,
-			expectedErr:      true,
+			fakeSecretLister:   failGetSecretLister,
+			fakeClient:         clientReturnsGenericError,
+			expectedErr:        true,
+			skipSecondSignCall: false,
 		},
 		"tpp: if sign returns cert then return cert and not failed": {
 			certificateRequest: tppCR.DeepCopy(),
 			builder: &controllertest.Builder{
 				KubeObjects:        []runtime.Object{tppSecret},
 				CertManagerObjects: []runtime.Object{tppCR.DeepCopy(), tppIssuer.DeepCopy()},
 				ExpectedEvents: []string{
+					"Normal IssuancePending Venafi certificate is requested",
 					"Normal CertificateIssued Certificate fetched from issuer successfully",
 				},
 				ExpectedActions: []testpkg.Action{
+					testpkg.NewAction(coretesting.NewUpdateSubresourceAction(
+						cmapi.SchemeGroupVersion.WithResource("certificaterequests"),
+						"status",
+						gen.DefaultTestNamespace,
+						gen.CertificateRequestFrom(tppCR,
+							gen.SetCertificateRequestStatusCondition(cmapi.CertificateRequestCondition{
+								Type:               cmapi.CertificateRequestConditionReady,
+								Status:             cmmeta.ConditionFalse,
+								Reason:             cmapi.CertificateRequestReasonPending,
+								Message:            "Venafi certificate is requested",
+								LastTransitionTime: &metaFixedClockStart,
+							}),
+							gen.AddCertificateRequestAnnotations(map[string]string{VenafiPickupIDAnnotation: "test"}),
+						),
+					)),
+					testpkg.NewAction(coretesting.NewUpdateSubresourceAction(
+						cmapi.SchemeGroupVersion.WithResource("certificaterequests"),
+						"",
+						gen.DefaultTestNamespace,
+						gen.CertificateRequestFrom(tppCR,
+							gen.SetCertificateRequestStatusCondition(cmapi.CertificateRequestCondition{
+								Type:               cmapi.CertificateRequestConditionReady,
+								Status:             cmmeta.ConditionFalse,
+								Reason:             cmapi.CertificateRequestReasonPending,
+								Message:            "Venafi certificate is requested",
+								LastTransitionTime: &metaFixedClockStart,
+							}),
+							gen.AddCertificateRequestAnnotations(map[string]string{VenafiPickupIDAnnotation: "test"}),
+						),
+					)),
```

Comment:
Same comment as above. This appears to be a duplicate except the "status" argument beomes "". Why? Maybe add an explanatory comment to the test.